src/webui/www/private/scripts/client.js

```diff
@@ -1308,7 +1308,7 @@ window.addEventListener("DOMContentLoaded", (event) => {
         document.getElementById("filtersColumn_handle").classList.add("invisible");
         document.getElementById("mainColumn").classList.add("invisible");
         document.getElementById("torrentsFilterToolbar").classList.add("invisible");
-        MochaUI.Desktop.resizePanels();
+        MochaUI.Desktop.setDesktopSize();
```

Comment:
We need to perform a full resize of the `pageWrapper`, otherwise the content won't expand properly. This can be seen on mobile, where the hiding of the toolbar results in more space for the main content (but the main content only expands to fill the space if the `pageWrapper` is resized). Note that `setDesktopSize()` eventually calls `resizePanels()`.